pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/FileUploadNotificationEmailFactory.java

```diff
@@ -7,85 +7,45 @@
 import lombok.NoArgsConstructor;
 import one.util.streamex.StreamEx;
 import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.exception.DDPException;
 import org.broadinstitute.ddp.model.files.FileUpload;
+import org.broadinstitute.ddp.model.mail.MailTemplateSubstitution;
+import org.broadinstitute.ddp.service.mail.MailTemplateService;
 
 import java.text.SimpleDateFormat;
 import java.util.Date;
 import java.util.List;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class FileUploadNotificationEmailFactory {
-    private static final SimpleDateFormat dateFormatter = new SimpleDateFormat("dd.MM.yyyy");
+    private static final SimpleDateFormat dateFormatter = new SimpleDateFormat("MM/dd/yyyy");
     private static final SimpleDateFormat timeFormatter = new SimpleDateFormat("HH:mm:ss");
 
     public static Mail create(final StudyDto study, final String userGuid, final List<FileUpload> fileUploads) {
+        if (study.getNotificationMailTemplateId() == null) {
+            throw new DDPException("Can't create e-mail. Study " + study.getGuid() + " doesn't have mail template");
+        }
+
+        final var mailTemplate = MailTemplateService.getTemplate(study.getNotificationMailTemplateId());
+        mailTemplate.setSubstitutions("PROJECT_SINGULAR_FILE_UPLOAD_ROW", StreamEx.of(fileUploads)
+                .map(fileUpload -> toSubstitution(userGuid, fileUpload))
+                .toList());
```

Comment:
References to specific studies do not belong in the codebase. This needs to be refactored to be for general use.

Reply:
I agree. Will rename the constant to something study-neutral.